models/ncnews.models.js

```diff
@@ -16,3 +16,29 @@ exports.selectSelectArticleById = (article_id) => {
       return rows[0];
     });
 };
+
+exports.selectArticles = () => {
+  return Promise.all([
+    db.query("SELECT * FROM articles ORDER BY created_at DESC;"),
+    db.query("SELECT * FROM comments;"),
+  ]).then((arrayOfTables) => {
+    if (arrayOfTables[0].length === 0) {
+      return Promise.reject({ status: 404, message: "Not found" });
+    }
+    const articles = arrayOfTables[0].rows;
+    const comments = arrayOfTables[1].rows;
+    articles.forEach((article) => {
+      delete article.body;
```

Comment:
rather than delete the body, you can just not select it in your SQL query. SQL is generally faster at handling queries so we should avoid JS manipulation if we can